regions/io/core.py

```diff
@@ -591,7 +605,10 @@ def to_shape_list(region_list, coordinate_system='fk5'):
     for region in region_list:
 
         coord = []
-        reg_type = str(type(region)).split(".")[2]
+        if isinstance(region, SkyRegion):
+            reg_type = region.__class__.__name__[:-9].lower()
+        else:
+            reg_type = region.__class__.__name__[:-11].lower()
```

Comment:
Is it really better to use these private classes rather than the string representations of the class names?  I'm a bit uncomfortable with this approach, even though I can't think of any reason not to use it.

Reply:
hmm, this just looks cleaner to me . If we use it's string representation , we need  lil more work to get the shape name.

Reply:
OK, you can leave this as is.